packages/truffle/test/scenarios/commands/console.js

```diff
@@ -0,0 +1,40 @@
+const CommandRunner = require("../commandRunner");
+const Server = require("../server");
+const Config = require("@truffle/config");
+const MemoryLogger = require("../MemoryLogger");
+const assert = require("assert");
+
+describe("truffle console", () => {
+  let config;
+  const logger = new MemoryLogger();
+
+  before(async () => await Server.start());
+  after(async () => await Server.stop());
+
+  before("before all setup", () => {
+    config = Config.default();
+    config.logger = logger;
+  });
+
+  describe("when run with --url option", () => {
+    it("displays the hostname in the prompt", async () => {
+      const url = "http://localhost:8545";
+      const parsedUrl = new URL(url);
+      const displayHost = parsedUrl.host;
+
+      await CommandRunner.runInREPL({
```

Comment:
If you include `inputCommands: []` it is clear the intention is to pass no input.

Reply:
Do we need to specify this, @cds-amal that we are not passing any `inputCommands` ? Leaving the parameter unspecified also means the same, right?

Reply:
It is the same thing to the interpreter, although, a human reader would benefit from the explicitness.

Reply:
Okay, that makes sense.